### PR TITLE
Add 'data:' URL scheme compat data

### PR DESCRIPTION
**Pull request by Elchi3**
If I see it correctly, this page is the last one in the MDN HTTP area that uses the old style (static) compat table :-)

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URIs

### CODE REVIEW

File: http/data-url.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+              },
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "12",
+                "notes": ["IE9 and later, as well as Edge, supports data URLs in CSS and JS files, but not in HTML files, with a max size of 4GB."]
```
Comment (wbamberg) [inline http/data-url.json]: I find this note hard to read :).

First, I have tried to avoid referring to browsers in the note itself, because it ends up reading weirdly (why is the Edge data talking about IE?).

Second, the complex run-on sentence makes it ambiguous and hard to parse. It could be read as: `data` is supported in CSS, JS, HTML. In CSS and JS the max size is 4GB.

It would be better written as: "data URLs can only be used in CSS and JS files. The maximum size supported is 4GB."

File: http/data-url.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+              "firefox": {
+                "version_added": true
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
+                "version_added": "8",
+                "notes": ["IE8 only supports data URLs in CSS files, with a max size of 32kB",
+                          "IE9 and later, as well as Edge, supports data URLs in CSS and JS files, but not in HTML files, with a max size of 4GB."]
```
Comment (wbamberg) [inline http/data-url.json]: These two notes make me wonder if it's worth using subfeatures here for "CSS files"/"JS files", "HTML files".

**Comment by Elchi3** [comment]
Thanks @wbamberg! I just copied the static table without thinking too much. But you are right, lets migrate this properly. I've split this into subfeatures and added notes about the size limit to the Edge and IE versions.
